schemas/blocks/interstitial.json

```diff
@@ -20,6 +20,25 @@
       "content": {
         "$ref": "https://eq.ons.gov.uk/blocks/definitions.json#/content"
       },
+      "individual_response": {
+        "type": "object",
+        "properties": {
+          "enabled": {
+            "type": "boolean",
+            "description": "Controls whether the individual response definition can be shown"
+          },
+          "for_list": {
+            "type": "string",
+            "description": "The list to use when determining whether the individual response definition can be shown"
+          },
+          "individual_section_id": {
+            "type": "string",
+            "description": "The id of the individual section for which an individual response can be requested"
+          }
+        },
+        "additionalProperties": false,
+        "required": ["enabled", "for_list", "individual_section_id"]
+      },
```

Comment:
We should move this outside interstitial and hub definitions and reference it to avoid the duplication.